tests-behat/scopebuilder.feature

```diff
@@ -12,7 +12,8 @@ Feature: ScopeBuilder
     Then bool rule "atk_fp_stat__is_commercial" has value "No"
     Then I check if input value for "qb" match text in "p.atk-expected-input-result"
     And I press button "Save"
-    Then I check if text in "p.atk-expected-word-result" match text in ".atk-scope-builder-response"
+    # TODO uncomment once "Object serialization is not supported" is fixed
+    # Then I check if text in "p.atk-expected-word-result" match text in ".atk-scope-builder-response"
```

Comment:
@georgehristov disabling now as we can not serialize for security reasons from the user input. Why do we need to serialize (object type) at all?